content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -870,7 +926,29 @@ public void processUnsubmitted() {
 					if ("true".equals(resource.getProperties().getProperty(AssignmentConstants.PROP_INLINE_SUBMISSION))
 							&& FilenameUtils.getExtension(fileName).isEmpty()) {
 						fileName += HTML_EXTENSION;
-					}							
+					}
+					
+					// Delete drafts that are not not submitted to TCA and flag drafts that are
+					boolean generateReportForDraft = false;
+					if(checkForDraft(item, assignment)) {
+						if("true".equals(assignment.getProperties().get("report_gen_draft"))) {
+							generateReportForDraft = true;
+						}
+						else {
+							// Drafts are not submitted, item can be deleted
+							crqs.delete(item);
```

Comment:
do not delete the draft item. Instead, we need to create a submission and placeholder but no report

Reply:
don't set status to pending so that the user doesn't see any flags